docs/BinSkimRules.md

```diff
@@ -226,6 +226,28 @@ No checked functions are present/used when compiling '{0}', and it was compiled
 
 ---
 
+## Rule `BA3031.EnableSafeStackWithClang`
```

Comment:
>EnableSafeStackWithClang [](http://example.com/codeflow?start=16&length=24)

EnableClangSafeStack?

I will think about the name and help refine this help text a bit further.

Reply:
fixed